portfolio/src/main/webapp/index.html

```diff
@@ -1,20 +1,48 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="about-content">
-        <div id = "about">
-            <div id="nav-bar">
-                <a href ="#about">about</a>
-                <a href ="#suggestions">suggestions</a>
-                <a href ="#map">map</a>
-            </div>
+
+<head>
+	<meta charset="UTF-8">
+	<title>My Portfolio</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
+<body>
+	<div id="about-content">
+		<div id="nav-bar">
+			<a href="#about">about</a>
+			<a href="#suggestions">suggestions</a>
+			<a href="#map">map</a>
+		</div>
+		<div id="intro">
+			<div id="intro-txt">
+				<div id="intro-txt-header"><span id = "highlight">hello</span></div>
+				<div id="intro-txt-paragraph">I'm <span id = "highlight">Sarah</span>, an aspiring software developer
+				</div>
+			</div>
+			<img src="images/devices-img.jpg" id = "intro-img"/>
+      </div>
```

Comment:
Can you try running the formatter on this file?

Reply:
Used the formatter tool on cloud shell but it indented wrong part way through but did a manual fix. However the file is appearing differently on github compared to what I see on cloud shell (https://screenshot.googleplex.com/3Sdz40k0pcq). Do I have to reformat on github rather than in cloud shell?